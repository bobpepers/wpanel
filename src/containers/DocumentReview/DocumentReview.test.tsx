import {shallow} from 'enzyme';
import * as React from 'react';
import {DocumentReview} from './';

describe('DocumentReview test', () => {
    it('should render', () => {
        const wrapper = shallow(<DocumentReview/>);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
