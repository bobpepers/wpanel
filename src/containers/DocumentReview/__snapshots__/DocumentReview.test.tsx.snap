// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentReview test should render 1`] = `
<Connect(withRouter(DocumentReviewComponent))
  classes={
    Object {
      "attachments": "Connect-withRouter-DocumentReviewComponent---attachments-8",
      "blueIcon": "Connect-withRouter-DocumentReviewComponent---blueIcon-6",
      "button": "Connect-withRouter-DocumentReviewComponent---button-10",
      "emptyTable": "Connect-withRouter-DocumentReviewComponent---emptyTable-9",
      "greenIcon": "Connect-withRouter-DocumentReviewComponent---greenIcon-4",
      "greyIcon": "Connect-withRouter-DocumentReviewComponent---greyIcon-3",
      "header": "Connect-withRouter-DocumentReviewComponent---header-1",
      "link": "Connect-withRouter-DocumentReviewComponent---link-2",
      "redIcon": "Connect-withRouter-DocumentReviewComponent---redIcon-5",
      "selectIcon": "Connect-withRouter-DocumentReviewComponent---selectIcon-7",
    }
  }
/>
`;
